compiler/shared_ast/operator.ml

```diff
@@ -332,88 +332,18 @@ let kind_dispatch :
     | Eq_mon_mon | Eq_dat_dat | Eq_dur_dur ) as op ->
     resolved op
 
+let translate (r : date_rounding option) t =
```

Comment:
`r` should be a optional argument with an explicit name, I read a call to `Operator.translate` earlier with a `None` and I had no idea what it meant.

Reply:
Indeed; actually I don't even think it's worth having it as a parameter to this function, it's unrelated and used only once.